lib/pharos/ssh_command.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Pharos
+  class SSHCommand < UpCommand
+    usage "[OPTIONS] -- [COMMANDS] ..."
+    parameter "[COMMAND] ...", "Run command on host"
+
+    banner "Opens SSH sessions to hosts in the Kontena Pharos cluster."
+
+    option ['-r', '--role'], 'ROLE', 'select a server by role'
+    option ['-l', '--label'], 'LABEL=VALUE', 'select a server by label, can be specified multiple times', multivalued: true do |pair|
+      Hash[*[:key, :value].zip(pair.split('=', 2))]
+    end
+    option ['-a', '--address'], 'ADDRESS', 'select a server by public address'
+
+    option ['-P', '--use-private'], :flag, 'connect to the private address'
```

Comment:
IMO private address is unnecessary because ssh is always done via `address`. What is the use case for this?